btk/catalog.py

```diff
@@ -70,7 +86,7 @@ def _prepare_table(self):
         # BTK now requires ref_mags, but Catsim still wants magnitudes
         table["ref_mag"] = self._raw_catalog["i_ab"]
         table["btk_size"] = r_sec
-        # Adds the extra columns to both catalogs just to be sure
+        # Adds the extra columns to both catalogs just to be sure - TO CHECK
```

Comment:
what needs to be checked exactly?

Reply:
More of reminder than actual checking ; I believe the two following lines are useless ; this will probably be tackled in the COSMOS PR though so adding the comment was not that useful